charts/fabric-operator/README.md

```diff
@@ -0,0 +1,112 @@
+# Fabric Operator Chart
+
+## Installation
+
+Quick start to deploy `FabricOperator` by `kubebb/core`.
+
+### Prerequisites
+
+Before installing fabric-operator, make sure that the `u4a-component` have been deployed.
+
+#### Install chart using ComponentPlan
+
+1. First create the Repository
+
+```yaml
+apiVersion: core.kubebb.k8s.com.cn/v1alpha1
+kind: Repository
+metadata:
+  name: sample-test
+  namespace: kubebb-system
+spec:
+  url: http://127.0.0.1:9999/def
+  pullStategy:
+    intervalSeconds: 120
+    retry: 5
+```
+
+2. Check if the component is created
+
+```shell
+$ kubectl get component.core.kubebb.k8s.com.cn -nkubebb-system
+NAME                                  AGE
+sample-test.fabric-operator           1m
+```
+
+3. Create ComponentPlan installation operator
+
+According to the contents of `values.yaml`, there are three positions that need to be replaced
+
+
+- namespace
+- iamServer
+- minio.host
+- minio.accesskey
+- minio.secretKey
+
+```yaml
+apiVersion: core.kubebb.k8s.com.cn/v1alpha1
+kind: ComponentPlan
+metadata:
+  name: fabric-operator
+  namespace: kubebb-system
+spec:
+  approved: true
+  name: fabric-operator
+  version: 0.1.0
+  override:
+    set:
+    - namespace=kubebb-system
```

Comment:
namespace is not required. we will use componentplan's namespace in the manifest